pom.xml

```diff
@@ -1304,14 +1306,14 @@
             <plugin>
                 <groupId>io.prestosql</groupId>
                 <artifactId>presto-maven-plugin</artifactId>
-                <version>6</version>
+                <version>7-SNAPSHOT</version>
```

Comment:
Is this required here?

Reply:
Keeps the version of provisio used by the presto-maven-plugin in sync with the version of provisio used to produce the presto-server archive. In this particular case there isn't an incompatibility as I adjusted a few things, but not changing this would mean presto would be using two different versions of provisio at least a year apart.

Reply:
Related PR https://github.com/prestosql/presto-maven-plugin/pull/4